src/js/main.js

```diff
@@ -42,24 +42,58 @@ function translate(text, sourceLang, targetLang, callback) {
 
   console.log('translate', sourceLang, targetLang);
 
-  var params = {
-    target: targetLang,
-    q: text
-  };
+  $.get(chrome.extension.getURL('api-proxy-sign'), function(signature) {
```

Comment:
This code has been changed to use the generic web API proxy. Again, see the `web-api-proxy` README for details.